### PR TITLE
feat: handle fully qualified service injections

### PR DESCRIPTION
**Pull request by cafreeman**
Implements https://github.com/rehearsal-js/rehearsal-js/issues/390

### CODE REVIEW

File: packages/migrate/src/migrate.ts [hunk 1]
```diff
@@ -101,13 +86,13 @@ export async function* migrate(input: MigrateInput): AsyncGenerator<string> {
...
   );
 
-  const { options, fileNames: someFiles } = parseJsonConfigFileContent(
-    config,
-    ts.sys,
-    dirname(configFile),
-    {},
-    configFile
-  );
+  const parsed = parseJsonConfigFileContent(config, ts.sys, dirname(configFile), {}, configFile);
```
Comment (egorio) [inline packages/migrate/src/migrate.ts:89]: @cafreeman  can we move this outside of migrating the package to keep it independent from any framework-specific details?

File: packages/plugins/src/plugins/index.ts [hunk 2]
```diff
@@ -5,3 +5,4 @@ export { PrettierPlugin, PrettierPluginOptions } from './prettier.plugin.js';
 export { ReRehearsePlugin, ReRehearsePluginOptions } from './re-rehearse.plugin.js';
 export { GlintCheckPlugin } from './glint-check.plugin.js';
 export { GlintFixPlugin } from './glint-fix.plugin.js';
+export { TransformPlugin } from './transform.plugin.js';
```
Comment (egorio) [inline packages/plugins/src/plugins/index.ts:8]: I think something like `ServiceInjectionsTransformPlugin` would be better :)

Comment (cafreeman) [reply packages/plugins/src/plugins/index.ts:8]: totally agree

File: packages/migrate/src/migrate.ts (hunk 1 above)
Comment (cafreeman) [reply packages/migrate/src/migrate.ts:89]: This actually existed prior to the glint stuff, but I did move all the glint-specific functions to a `glint-utils` file like we discussed

File: packages/migrate/test/migrate.test.ts [hunk 3]
```diff
@@ -166,6 +166,39 @@ export default class Hello extends Component {
       expectFile(outputs[0]).toEqual(expected);
     });
 
+    test('with service', async () => {
```
Comment (chadhietala) [inline packages/migrate/test/migrate.test.ts:169]: @cafreeman I know the task is to deal with namespaced services but can you add a test case for non namespaced services to make sure what we get as output is sane?

**Review by lynchbomb — APPROVED** [review]

**Pull request merged**